create-new-data-pr.py

```diff
@@ -68,7 +67,10 @@ def get_tag_from_string(tag_string=None):
   # if created files and modified files are the same count, all files are new
 
   response = urlopen("https://api.github.com/repos/%s/pulls/%s" % (opts.data_repo, opts.pull_request))
-  res_json = loads(response.read())
+  res = response.read()
+  if isinstance(res, bytes):
+    res = res.decode()
+  res_json = loads(res)
```

Comment:
@iarspider , do we need this is script runs under python3?
by the way, you need to update the `shebang` to use python3 ... right?

Reply:
This script is used by https://cmssdt.cern.ch/jenkins/job/update-data-tag-on-pr-merge/, which runs on cmssdt nodes. I will update hashbang, thanks for noticing.

Reply:
I mean do we need this logic https://github.com/cms-sw/cms-bot/pull/2066/files#diff-d401cb5613b47d8eca73383d19d96514e9af6681747d2bcd461c4d6812c334aaR70-R73 if script only runs under python3?

Reply:
Technically, no: `urlopen`, I think, now always return `bytes`.